_posts/2024-12-15-kob-epa-fuer-alle.markdown

```diff
@@ -0,0 +1,55 @@
+
+With the rollout of the 'ePA für alle', a new approach has been introduced: the **KOB Test Suite**, based on Tiger. Tiger, developed by gematik, is a test framework designed for rapid, transparent, and efficient test suites. The KOB Test Suite is publicly available on GitHub, free of charge, allowing manufacturers to download and execute it locally early in their development process. This supports integration into existing CI pipelines and use on individual developer machines. Test results can be uploaded to the confirmation portal, where the largely automated review process certifies conformity in a short time.
+
+## A Paradigm Shift in Testing
+
+This new process offers significant advantages for all parties involved: faster public availability of test suites, early support during development, precise and transparent test results, and flexible integration into internal workflows. The iterative approach fosters better collaboration between gematik and manufacturers, enabling more efficient development.
+
+By entrusting testing to manufacturers while maintaining complete transparency regarding test content, gematik builds trust. The open-source nature of the test suite ensures all code can be reviewed and understood. Early testing (a "shift-left" approach) further supports manufacturers by improving time-to-market outcomes.
+
+## The Tiger Framework
+
+Tiger is a comprehensive test solution designed to simplify the creation of various test types, including end-to-end, acceptance-, integration-, and interface tests. Ideal for resource-constrained teams and agile processes, it is open-source, Java-based, and built on Cucumber, offering flexibility and customizability for automated testing.
+
+Tiger’s core feature is a proxy that intercepts and analyzes network traffic between components. This traffic is converted into a protocol-agnostic hierarchical structure (a tree), searchable with **RbelPath**, a query language inspired by XPath and JsonPath. This eliminates the need for advanced programming skills, making the framework accessible even for non-technical users.
+
+To enhance collaboration, Tiger provides the **WorkflowUI**, an HTML-based visualization of captured traffic, simplifying documentation and communication around test scenarios. The framework's plugin-based architecture supports extensions tailored to specific needs, such as Docker container provisioning or FHIR resource validation. Additionally, **Zion** enables the rapid deployment of mock services with simple logic.
+
+## KOB Tests with Tiger
+
+The KOB Test Suite, available on GitHub, allows manufacturers to run tests in their environments, providing significant flexibility. Internal setups can be used, and tests can be executed on individual developer machines or central build servers. Test results are immediate and actionable.
```

Comment:
Maybe  also a link to the GitHub Repository?

Reply:
done